code/game/objects/items/melee/baton.dm

```diff
@@ -559,9 +566,14 @@
 			addtimer(CALLBACK(src, .proc/scramble_mode), scramble_time*loops * (1 SECONDS))
 
 /obj/item/melee/baton/security/proc/scramble_mode()
+<<<<<<< Updated upstream
 	active = !active
 	playsound(src, "sparks", 75, TRUE, -1)
 	update_appearance()
+=======
+	set_active(!active)
+	playsound(src, "sparks", 75, TRUE, -1)
+>>>>>>> Stashed changes
```

Comment:
You still have leftovers from a failed merge/stash that are breaking things.

Reply:
where exacly and how to fix it?

Reply:
nevermind I am pretty dense